scss/components/_case-tab.scss

```diff
@@ -0,0 +1,7 @@
+.civicase__case-tab__container {
```

Comment:
Where are these used?

Reply:
In civiawards:
https://github.com/compucorp/uk.co.compucorp.civiawards/pull/44/files#diff-ff25b4481cf5cfbb467ba72d9318702bR1-R2

In civicase:
https://github.com/compucorp/uk.co.compucorp.civicase/pull/317/files#diff-ea42e21a650a48c436cf5610b9b48c8bR1